src/system_variables.star

```diff
@@ -12,8 +12,8 @@ NODE_CONFIG_FILE_LOCATION = "github.com/logos-co/wakurtosis/config/topology_gene
 
 CONTAINER_NODE_CONFIG_FILE_LOCATION = "/node/configuration_file/"
 NODE_CONFIGURATION_FILE_EXTENSION = ".toml"
-NWAKU_ENTRYPOINT = ["/usr/bin/wakunode", "--rpc-address=0.0.0.0", "--metrics-server-address=0.0.0.0"]
-GOWAKU_ENTRYPOINT = ["/usr/bin/waku", "--rpc-address=0.0.0.0", "--metrics-server-address=0.0.0.0"]
+NWAKU_ENTRYPOINT = ["/usr/bin/wakunode", "--rpc-address=0.0.0.0", "--metrics-server-address=0.0.0.0", "--store=true", "--storenode=/dns4/node_0"]
```

Comment:
Uhm, `--store=true` here for sure it's not an issue but I would not put it here (both in nwaku and gowaku) because this will activate by default store in everything, and the idea is that we should activate this in the `toml` file.
And I think `--storenode=/dns4/node_0` should be in `toml` file aswell.
Basically here are the flags that are "mandatory", which are runing the node, and accepting incoming calls from everywhere so it works with docker network. The rest of the flags should not be here imo.

Reply:
So you are suggesting to add this in Gennet?

Reply:
I think  what alberto is suggesting  is that these configs be specified in gennet's per-node toml outputs.

Reply:
Ok, I will open an issue